### ISSUE TITLE
Separate output for dmrs?

### ISSUE THREAD

**Issue by mengzhou** [smithlabcode/radmeth#2]
The program `dmrs` calls DMR from table of adjusted pvalue. However the output is mixed with DMRs in both directions, e.g. dmr:(some positive value) and dmr:(some negative value), and user has to manually separate DMRs from the result.

For myself I rarely use DMRs of both directions in one file, so I suggest it's better to use two options for output.

**Comment by egor-dolzhenko** [comment]
Good point. I'll modify the `dmr` program accordingly as soon as we hear back from the reviewers. I probably shouldn't change anything until then.

**Comment by andrewdavidsmith** [comment]
You can do it before if you document properly. I'm not sure if this is a
bad thing, though. Typically we do not want 2 output files for each run of
a program.

**Comment by egor-dolzhenko** [comment]
There is another potential issue with splitting the output. When there are sufficiently many samples, RADMeth will flag regions with very variable methylation in one group of samples and consistent methylation in the other (e.g. a bimodal distribution in one group and a binomial in the other). There are many biological contexts where such behavior is interesting and, I believe, expected (e.g. cancer vs non-cancer comparisons). It does not really make sense to talk about directionality in such situations.

**Comment by pjuren** [comment]
As a general rule, I try to avoid outputting multiple files as the default behavior -- it makes writing pipelines that use the tool a bit more tricky. As a suggested alternative, you could add an option that changes the behavior so that only DMRs in a particular direction are output. That would allow use-cases like Meng's to be easily accommodated, without having to change the current default behavior.
